src/common/identity_parser/identity_parser.cpp

```diff
@@ -37,6 +39,39 @@ kvp::ExpectedKeyValuesMap GetIdentityData(const string &identity_data_generator)
 	return ex_key_values;
 }
 
+string DumpIdentityData(kvp::KeyValuesMap identity_data) {
+	stringstream top_ss;
+	top_ss << "{";
+	auto key_vector = common::GetMapKeyVector(identity_data);
+	std::sort(key_vector.begin(), key_vector.end());
+	for (const auto &key : key_vector) {
+		top_ss << json::EscapeString(key);
+		top_ss << R"(:)";
```

Comment:
The use of the raw string on this line is a big confusing,it's unnecessary, right?

Reply:
It is not necessary.

Ironically, I thought it would be _less_ confusing if I used raw strings consistently instead of combining the two typings :thinking: